### PR TITLE
Added Kungfu Panda

### PR DESCRIPTION
**Pull request by ashwinshawntauro**
Added:
- Card: Kungfu Panda 
- Theme: pandaLight, pandaDark

### CODE REVIEW

File: src/data/kungfu_panda_quotes.json
Comment (akanshSirohi) [inline src/data/kungfu_panda_quotes.json:1]: @ashwinshawntauro Please add at-least 100 entries to your card.

File: src/themes.js
Comment (akanshSirohi) [inline src/themes.js:1]: @ashwinshawntauro Please try to change the required code only, modifying the whole code is not necessary.

**Review by akanshSirohi — CHANGES_REQUESTED** [review]
@ashwinshawntauro Please do the changes requested.

File: src/themes.js
Comment (ashwinshawntauro) [reply src/themes.js:1]: @akanshSirohi Only changed card details and theme

Comment (akanshSirohi) [reply src/themes.js:1]: @ashwinshawntauro You still have not fixed this file, you have added alot of changes which I am unable to check. Please add only required changes so I can verify. Please do not format whole file.

**Review by akanshSirohi — CHANGES_REQUESTED** [review]
@ashwinshawntauro Please do the requested changes

**Comment by akanshSirohi** [comment]
@ashwinshawntauro Also please resolve conflicts in your branch

**Pull request closed without merging**